OPHD/States/MapViewStateUi.cpp

```diff
@@ -427,13 +438,31 @@ void MapViewState::btnToggleCommRangeOverlayClicked()
 	if (mBtnToggleCommRangeOverlay.toggled())
 	{
 		mBtnToggleConnectedness.toggle(false);
+		mBtnToggleRouteOverlay.toggle(false);
+
 		btnToggleConnectednessClicked();
+		btnToggleRouteOverlayClicked();
 	}
 
 	setOverlay(mBtnToggleCommRangeOverlay, mCommRangeOverlay, Tile::Overlay::Communications);
 }
 
 
+void MapViewState::btnToggleRouteOverlayClicked()
+{
+	if (mBtnToggleRouteOverlay.toggled())
+	{
+		mBtnToggleConnectedness.toggle(false);
+		mBtnToggleCommRangeOverlay.toggle(false);
+
+		btnToggleCommRangeOverlayClicked();
+		btnToggleConnectednessClicked();
+	}
```

Comment:
Kind of wondering if we might need some kind of `ToggleButtonGroup` control, which groups several options together, and tracks which one is currently active. It can be responsible for calling a `deactivate` method on the previously active button, and an `activate` method on the newly active button.

Reply:
Would make a lot of sense since we can also group check boxes and radio buttons in the same way.